Distinguish between the cluster in docs and examples (#2741)
CrunchyData/postgres-operator: docs/content/tutorial/connect-cluster.md

## Connect an Application

For this tutorial, we are going to connect [Keycloak](https://www.keycloak.org/), an open source
identity management application. Keycloak can be deployed on Kubernetes and is backed by a Postgres

> database. While we provide an [example of deploying Keycloak](https://github.com/CrunchyData/postgres-operator-examples/tree/main/kustomize/keycloak)
> in the [Postgres Operator examples](https://github.com/CrunchyData/postgres-operator-examples)
> repository, we will use the manifest below to deploy the application using our `hippo` cluster:

Author: ```suggestion
database. While we provide an [example of deploying Keycloak and a PostgresCluster](https://github.com/CrunchyData/postgres-operator-examples/tree/main/kustomize/keycloak)
in the [Postgres Operator examples](https://github.com/CrunchyData/postgres-operator-examples)
repository, the manifest below deploys it using our `hippo` cluster that is already running:
```

Reviewer: I like the suggestion :+1: 

Think that def makes it even clearer.
